src/server/game/Entities/Player/Player.cpp

```diff
@@ -9324,6 +9324,13 @@ void Player::SendInitWorldStates(uint32 zoneid, uint32 areaid)
                 data << uint32(4882) << uint32(0);              // 10 WORLD_STATE_HOR_WAVE_COUNT
             }
             break;
+        case 85:    // Tirisfal Glades
+        case 1497:  // Undercity
+        {
+            if (LocationScript* location = sObjectMgr->GetLocationScript(85))
+                location->FillInitialWorldStates(data);
```

Comment:
WorldStates are limited to players in phases only, so they won't see it, unless you're talking about performance then ill bool it later